optuna/pruners/hyperband.py

```diff
@@ -80,6 +80,12 @@ class HyperbandPruner(BasePruner):
             The number of :class:`~optuna.pruners.SuccessiveHalvingPruner`\\ s (brackets).
             Defaults to :math:`4`.
         min_early_stopping_rate_low:
+
+            .. deprecated:: 1.4.0
+                This argument will be removed from :class:~optuna.pruners.HyperbandPruner. The
+                 minimum value of the minimum early-stopping rate are automatically determined
```

Comment:
Are both "minimum" necessary here?

Reply:
Yes, I think it is necessary, because the `minimum early-stopping rate` is determined for each bracket and the `min_early_stopping_rate_low` is the minimum value among those. But it looks strange sentence... Please give any idea if possible.